spec/lib/aba/batch_spec.rb

```diff
@@ -39,23 +53,26 @@
     context 'when detail record' do
 
       it "should contain transactions records" do
-        expect(subject.to_s).to include("1342-342  3244654 530000000030John Doe                        R435564           453-543 45656733Remitter        00000000\r\n")
-        expect(subject.to_s).to include("1342-342  3244654 530000000020John Doe                        R435564           453-543 45656733Remitter        00000000\r\n")
+        expect(batch.to_s).to include("1342-342  3244654 530000000030John Doe                        R435564           453-543 45656733Remitter        00000000\r\n")
+        expect(batch.to_s).to include("1342-342  3244654 530000000020John Doe                        R435564           453-543 45656733Remitter        00000000\r\n")
+        expect(batch.to_s).to include("2453-543 456567338530000000003John Doe                        R435564           342-342  3244654Remitter        12654321\r\n")
+        expect(batch.to_s).to include("2453-543 456567338530000000002John Doe                        R435564           342-342  3244654Remitter        12654321\r\n")
```

Comment:
Nit pick, instead of multiple expectations, consider a single expectation using something like the squiggly heredoc.

Also, the `it` description isn't exactly accurate as the resulting expectation is that there are two type 1 records (DD/DC transaction) and two type 2 records (returns)

Reply:
Squiggly heredoc isn't supported until Ruby 2.3 but the gem still supports 1.9.2 so that's not an option.

I've updated the `it` description though. 👍

Reply:
Also, this currently specifically tests for `\r\n` where multiline squiggly heredoc would contain `\n` (probably dependent on your editor).

I'm guessing that we'd be better off just leaving this alone.